### ISSUE TITLE
no eth please return btc to 17Bix4JPxkWq8xsFeCGCGn2gokqWM5SbeW or 0xff86Ca9f9b5f11dc447Ec94bdDAc188F7d77D07A 35hrs + now

### ISSUE THREAD

**Issue by grtt** [ethereum/mist#2084]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by garretlaxton** [comment]
Nobody has access to your ETH but yourself, you need to let your wallet fully sync before you can see/send your ETH.

**Comment by grtt** [comment]
Hi don't have access to  account now

**Comment by grtt** [comment]
Hi can not access account now

**Comment by grtt** [comment]
12KDKaWRPSN6CZ6U93pSjxnzi6PD1PaiCK

**Comment by luclu** [comment]
Closed as duplicate of #2070.

**Comment by grtt** [comment]
no more messages
